CMakeLists.txt

```diff
@@ -76,3 +80,34 @@ if (RC_ENABLE_EXAMPLES)
 endif()
 
 add_subdirectory(extras)
+
+# install the targets and include directories
+install(TARGETS rapidcheck
+	ARCHIVE DESTINATION "${CMAKE_INSTALL_PREFIX}/lib"
```

Comment:
My apologies. I thought vim had the right syntax loaded...I'll fix the tabs. As far as your continuations: what mean you?